radio/src/targets/horus/CMakeLists.txt

```diff
@@ -155,6 +156,19 @@ elseif (PCB STREQUAL X12S)
   add_definitions(-DMANUFACTURER_FRSKY)
 endif()
 
+#define the number of allowed flexible adc entries used as switch
+if(FLEXSW EQUAL "1")
+  add_definitions(-DMAX_FLEX_SWITCHES=1)
```

Comment:
Is there a particular reason for not just passing the CMake option `as-is` (basically `${FLEXSW}`), and checking it's validity per `static_assert` in `hal/switch_driver.cpp`?